src/zzz_od/api/routers/random_play.py

```
@@ -0,0 +1,199 @@
+    ```
+    """
+    ctx = get_ctx()
+    record = ctx.random_play_run_record
+    return {
+        "finished": record.finished,
+        "timeCost": record.time_cost,
+    }
+
+
+def _run_via_onedragon_with_temp(app_ids: list[str]) -> str:
+    """通过一条龙总控运行指定 appId 列表（临时运行清单）。"""
```

Ruff (0.12.2)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)